app/preprints/-components/submit/author-assertions/template.hbs

```diff
@@ -0,0 +1,3 @@
+<div local-class='preprint-input-container  {{if (is-mobile) 'mobile'}}'>
```

Comment:
No need for this wrapper div if it has the same styling as the wrapper div in submit/template.hbs